docs/installationguide/debian.rst

```diff
@@ -154,314 +154,3 @@ Reload Apache to enable your changes and then start ZoneMinder.
     sudo systemctl start zoneminder
 
 You are now ready to go with ZoneMinder. Open a browser and type either ``localhost/zm`` one the local machine or ``{IP-OF-ZM-SERVER}/zm`` if you connect from a remote computer.
-
-Easy Way: Debian Stretch
```

Comment:
Did you mean to delete Stretch? Only mentioned Jessie.

Reply:
Whoops. Good catch.